### PR TITLE
Update commnity boolean field name

### PR DESCRIPTION
**Pull request by mcass19**

### CODE REVIEW

File: lib/oli/groups/community.ex [hunk 1]
```diff
@@ -6,15 +6,15 @@ defmodule Oli.Groups.Community do
     field :name, :string
     field :description, :string
     field :key_contact, :string
-    field :prohibit_global_access, :boolean
+    field :global_access, :boolean, default: true
```
Comment (jrissler) [inline lib/oli/groups/community.ex:9]: Add null: false to protect against nil please

File: test/oli/groups_test.exs [hunk 2]
```diff
@@ -14,12 +14,12 @@ defmodule Oli.GroupsTest do
...
           description: "Testing description",
           key_contact: "Testing key contact",
-          prohibit_global_access: false})
+          global_access: false})
 
       assert community.name == "Testing name"
       assert community.description == "Testing description"
       assert community.key_contact == "Testing key contact"
-      assert community.prohibit_global_access == false
+      assert community.global_access == false
```
Comment (jrissler) [inline test/oli/groups_test.exs]: I would like to see use of the factory here to remove hard-coded assertions such as:

`
params = params_for(:community)
assert {:ok, %Community{} = community} = Groups.create_community(params)

assert community.name == params.name
etc
etc
`

File: test/oli_web/live/community_live_test.exs [hunk 3]
```diff
@@ -178,7 +178,7 @@ defmodule OliWeb.CommunityLiveTest do
                  name: "Testing name",
                  description: "Testing description",
                  key_contact: "Testing key contact",
-                 prohibit_global_access: true
+                 global_access: true
```
Comment (jrissler) [inline test/oli_web/live/community_live_test.exs]: see comment above - think you can also use params_for to pull values from factory vs hard-coded values here.

**Review by jrissler — APPROVED** [review]

**Review by gastonabella — APPROVED** [review]
LGTM! 🚀

File: lib/oli/groups/community.ex (hunk 1 above)
Comment (mcass19) [reply lib/oli/groups/community.ex:9]: Added in the migration. What we can do here is call `validate_required` in the changeset function if that works for you

Comment (jrissler) [reply lib/oli/groups/community.ex:9]: No need it will always have a value.

**Pull request merged**